src/Frontend/Modules/Mailmotor/Domain/Subscription/SubscribeType.php

```diff
@@ -76,7 +76,7 @@ public function buildForm(FormBuilderInterface $builder, array $options): void
             'subscribe',
             SubmitType::class,
             [
-                'label' => 'lbl.Subscribe',
+                'label' => ucfirst(Language::lbl('Subscribe')),
```

Comment:
wrong place to fix this, this should be, if needed fixed in the form templates not here